magpie/ui/home/templates/template.mako

```diff
@@ -1,11 +1,20 @@
 <!DOCTYPE HTML>
-<html>
+<html lang="en">
 <head>
-    <title>Magpie Administration</title>
+    <title>Magpie</title>
     <meta charset="UTF-8">
     <link rel="stylesheet" type="text/css" href="//fonts.googleapis.com/css?family=Open+Sans" />
-    <link rel="shortcut icon" type="image/x-icon" href="${request.static_url('magpie.ui.home:static/settings.png')}" />
-    <link href="${request.static_url('magpie.ui.home:static/style.css')}" rel="stylesheet" type="text/css" media="all" />
+    <link href="${request.static_url('magpie.ui.home:static/settings.png')}"
+          rel="shortcut icon" type="image/x-icon" />
+    <link href="${request.static_url('magpie.ui.home:static/themes/{}.css'.format(MAGPIE_UI_THEME))}"
```

Comment:
You really provide a back-compatible theme! :)

Reply:
Yes :D